packages/babel-plugin-transform-merge-sibling-variables/__tests__/transform-merge-sibling-variables-test.js

```diff
@@ -36,4 +36,33 @@ describe("transform-merge-sibling-variables-plugin", () => {
 
     expect(transform(source).trim()).toBe(expected);
   });
+
+  it("don't concat block-scoped variables in for loops", () => {
+    const source = unpad(`
+      let i = 0;
+      for (let x = 0; x < 10; x++) console.log(i + x);
+    `);
+    const expected = unpad(`
+      let i = 0;
+      for (let x = 0; x < 10; x++) console.log(i + x);
+    `);
+
+    expect(transform(source).trim()).toBe(expected);
```

Comment:
Do we just reuse the source if it's the same? Forgot what other tests have done (in this case you could `expect(transform(source).trim()).toBe(source);`